client/src/components/Routes/Private.js

```diff
@@ -1,20 +1,28 @@
-import { useState,useEffect } from "react";
+import React, { useState,useEffect} from "react";
 import { useAuth } from "../../context/auth";
 import { Outlet } from "react-router-dom";
 import axios from 'axios';
-import { set } from "mongoose";
 import Spinner from "../Spinner";
 
 export default function PrivateRoute(){
     const [ok,setOk] = useState(false)
-    const [auth,setAuth] = useAuth()
+    const [auth] = useAuth()
 
     useEffect(()=> {
         const authCheck = async() => {
-            const res = await axios.get("/api/v1/auth/user-auth");
-            if(res.data.ok){
-                setOk(true);
-            } else {
+            try {
+                const res = await axios.get("/api/v1/auth/user-auth", {
+                    headers: {
+                        Authorization: auth?.token
+                    }
+                });
+                if(res.data.ok){
+                    setOk(true);
+                } else {
+                    setOk(false);
```

Comment:
Maybe can simplify to
```suggestion
                setOk(res.data.ok);
```